Update to jakarta common annotations 2.1.1 to pick up the deprecated …

starksm64 opened the pull request:
…@ManagedBean

Needed for EE 10 signature tests

starksm64 committed:
Update to jakarta common annotations 2.1.1 to pick up the deprecated @ManagedBean

bstansberry commented:
A few of the CI jobs failed because it looks like the artifact wasn't available when they tried to pull it. Others are running ok so I restarted those.

starksm64 committed:
Actually update preview.version.jakarta.activation.jakarta.activation-api

The previous change was to preview.version.jakarta.activation.jakarta.activation-api

starksm64 commented:
Actually, the update was incorrectly applied to the preview.version.jakarta.activation.jakarta.activation-api property. I updated the PR to set preview.version.jakarta.annotation.jakarta-annotation-api to 2.1.1.

starksm64 commented:
It is strange that the dependency tree check passed with the bad activation version.
